docs/fundamentals/networking/telemetry/metrics.md

```diff
@@ -26,122 +26,48 @@ There are two parts to using metrics in a .NET app:
 
 This section demonstrates various methods to collect and view System.Net metrics.
 
-### Example app
+### .NET Aspire
```

Comment:
Are we sure we want to entirely replace this paragraph instead of eg. only replacing the prometheus+grafana part with the aspire example? The original example app is simple and self-contained and the paragraph also demonstrates `dotnet-monitor`.